edge-middleware/bot-protection-datadome/lib/datadome.ts

```diff
@@ -200,11 +218,21 @@ function stringify(obj: Record<string, string | number | null | undefined>) {
         .join('&')
     : ''
 }
-
-function getCookiesLength(cookies: NextRequest['cookies']) {
+/**
+ * Returns a simple object with two properties:
+ *   - The client ID from the `datadome` cookie.
+ *   - The total length of the `Cookie` request header.
+ * @param {NextRequest['cookies']} cookies - Incoming client request cookie header
+ * @returns {{ clientId: string, cookiesLength: number }}
+ */
+function getCookieData(cookies: NextRequest['cookies']) {
+  let clientId = ''
   let cookiesLength = 0
   for (const [, cookie] of cookies) {
     cookiesLength += cookie.value.length
```

Comment:
This doesn't take into account the length of the name and the `=`, does it? We should send the total length of the cookies.